dfencoder/autoencoder.py

```diff
@@ -621,26 +631,48 @@ def compute_baseline_performance(self, in_, out_):
         return net_loss
 
     def _create_stat_dict(self, a):
-        scaler = StandardScaler()
+        scaler = self.loss_scaler()
         scaler.fit(a)
-        mean = scaler.mean
-        std = scaler.std
-        return {'scaler': scaler, 'mean': mean, 'std': std}
```

Comment:
Removing `mean` and `std` here will cause error in this part of `DFPMLFlowModelWriterStage`:

https://github.com/nv-morpheus/Morpheus/blob/branch-23.01/examples/digital_fingerprinting/production/morpheus/dfp/stages/dfp_mlflow_model_writer.py#L185-L188

These values saved to MLflow don't appear to be pulled and used anywhere in Morpheus. DFP training and inference worked fine with this `dfencoder` branch after removing the above lines.